app-catalog/src/settings.tsx

```diff
@@ -0,0 +1,96 @@
+import { HoverInfoLabel, NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Box, Switch, SwitchProps, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+
+export function AppCatalogSettings() {
+  const [showVerified, setShowVerified] = useState<boolean>(() =>
+    JSON.parse(localStorage.getItem('showVerified') ?? 'true')
+  );
+
+  function toggleShowVerified() {
+    setShowVerified(!showVerified);
+    localStorage.setItem('showVerified', `${!showVerified}`);
+  }
+
+  return (
+    <Box
+      style={{
+        marginTop: '3rem',
+      }}
+    >
+      <Typography variant="h5">App Catalog Settings</Typography>
+      <NameValueTable
+        rows={[
+          {
+            name: (
+              <HoverInfoLabel
+                label="Show Verified"
```

Comment:
Please rename this as `Only verified`. Otherwise it seems like we will stop showing verified charts.